src/go/wsl-helper/cmd/dnsmonitor.go

```diff
@@ -0,0 +1,47 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/wsl-helper/pkg/dnsmonitor"
+)
+
+// dnsmonitorCmd represents the dnsmonitor command
+var dnsmonitorCmd = &cobra.Command{
+	Use:   "dns-monitor",
+	Short: "Monitor to determine correct DNS servers",
+	Long:  `This monitor observes for the most appropriate DNS servers when VPN is used with WSL2 and updates resolve.conf accordignly`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		dnsFilePath, _ := cmd.Flags().GetString("path")
+		if dnsFilePath == "" {
+			cmd.Help()
+			os.Exit(0)
+		}
+		return dnsmonitor.Run(dnsFilePath)
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(dnsmonitorCmd)
+	dnsmonitorCmd.Flags().StringP("path", "p", "", "Path to /etc/resolve.conf")
```

Comment:
If we set the flag as [required](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagRequired) we don't need to handle it being missing (I think?)